.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 npm-debug.log
 compiler.jar
 contrib/
+jscomp.js
```

Comment:
Can we narrow this to the particular output directory, or just ignore the output directory containing it?  Like `/jscomp.js` or `/compiler/target/**/jscomp.js` or `/compiler/target/` if I'm reading build_compiler.js:70 right.

This seems like the kind of thing that might inadvertently match a source file in the future.